frontend/packages/console-app/src/components/guided-tours/utils/guided-tour-status.ts

```diff
@@ -0,0 +1,5 @@
+export enum GuidedTourStatus {
+  complete = 'Complete',
+  inProgress = 'In progress',
+  notStarted = 'Not started',
```

Comment:
I need to find an existing lint rule for this or create a new one. But we should prefer uppercase literal names.
```suggestion
  COMPLETED = 'Complete',
  IN_PROGRESS = 'In progress',
  NOT_STARTED = 'Not started',
```

Reply:
updated